src/services/completions.ts

```diff
@@ -5836,8 +5839,23 @@ function isProbablyGlobalType(type: Type, sourceFile: SourceFile, checker: TypeC
     return false;
 }
 
-function isStaticProperty(symbol: Symbol) {
-    return !!(symbol.valueDeclaration && getEffectiveModifierFlags(symbol.valueDeclaration) & ModifierFlags.Static && isClassLike(symbol.valueDeclaration.parent));
+function isStaticProperty(symbol: Symbol) {
+    return !!(symbol.valueDeclaration && getEffectiveModifierFlags(symbol.valueDeclaration) & ModifierFlags.Static && isClassLike(symbol.valueDeclaration.parent));
+}
+
+function isNativeFunctionMethod(symbol: Symbol): boolean {
+    const declaration = symbol.valueDeclaration ?? symbol.declarations?.[0];
+    if (!declaration) {
+        return false;
+    }
+    
+    const parent = declaration.parent;
+    if (!isInterfaceDeclaration(parent)) {
+        return false;
+    }
+    
+    const interfaceName = parent.name?.text;
+    return interfaceName === "Function" || interfaceName === "CallableFunction";
```

Comment:
This is not a comment to take an action on, but, why doesn't this check the global scope thing you mentioned previously?